exporter/kinesisexporter/exporter.go

```diff
@@ -52,8 +53,7 @@ func newKinesisExporter(c *Config, logger *zap.Logger) (*Exporter, error) {
 		return nil, err
 	}
 	client := kinesis.New(sess)
-
-	pr := producer.New(&producer.Config{
+	var pr = producer.New(&producer.Config{
```

Comment:
Out of curiosity, why var pr vs pr :=?

Reply:
Woops, that wasn't supposed to be there.